builder/virtualbox/common/comm_config.go

```diff
@@ -26,6 +26,9 @@ type CommConfig struct {
 	// does not setup forwarded port mapping for communicator (SSH or WinRM) requests and uses ssh_port or winrm_port
 	// on the host to communicate to the virtual machine.
 	SkipNatMapping bool `mapstructure:"skip_nat_mapping" required:"false"`
+	// Defaults to false. When enabled, the ssh port forwarding will be set to listen on 0.0.0.0
+	// as opposed to 127.0.0.1
+	SSHListenOnAllInterfaces bool `mapstructure:"ssh_listen_on_all_interfaces" required:"false"`
```

Comment:
I wonder if we could specify the listening address to be whatever we choose here instead of this flag?

Reply:
Hi @lbajolet-hashicorp,

Thanks again for the comments. I have pushed a new commit.